sdcm/nemesis.py

```diff
@@ -366,7 +366,10 @@ def _destroy_data_and_restart_scylla(self):
             for _ in range(5):
                 file_for_destroy = self._choose_file_for_destroy(ks_cfs)
 
-                result = self.target_node.remoter.run('sudo rm -f %s' % file_for_destroy)
+                if corrupt_by_dd:
+                    result = self.target_node.remoter.run('sudo dd if=/dev/random of=%s count=1024' % file_for_destroy)
```

Comment:
better explicitly mention block size (bs)

Reply:
question: why did you choose to create a 512k file?

Reply:
He just write some bytes into an sstable in order to corrupt it.